crates/sozo/src/ops/migration/mod.rs

```diff
@@ -238,7 +251,7 @@ where
 
                 let _ = TransactionWaiter::new(transaction_hash, migrator.provider())
                     .await
-                    .map_err(MigrationError::<S, <P as Provider>::Error>::WaitingError);
+                    .map_err(|_| anyhow!("Transaction execution failed"))?;
```

Comment:
converted this to `anyhow` error because previously we would be just ignoring this error, not sure why it was written that way.

Reply:
refer to comment above